Makefile

```diff
@@ -137,9 +138,10 @@ generate-py:  ## Generate .py files from notebooks
 
 test-tutorial: generate-py pip-install-tutorial ## Verify that all tutorials run without errors
 	echo "You must install the pip venv first. Run make pip-install-tutorial."
-	SYSTEM=$$(uname -s)
-	if [ $$SYSTEM = "Darwin" ]; then XARGS=gxargs; else XARGS=xargs; fi
-	ls notebooks/Tutorial*.py | xargs -n1 basename | $$XARGS -I {{}} bash -eu -o pipefail -c "cd notebooks/ && AVATAR_BASE_URL=$(AVATAR_BASE_URL) AVATAR_USERNAME=$(AVATAR_USERNAME) AVATAR_PASSWORD=$(AVATAR_PASSWORD) $(abspath $(VENV_NAME))/bin/python3.10 {{}} > /dev/null && echo \'Succesfully ran {{}}\'"
+	cd notebooks && ls Tutorial*.py | xargs -n1 basename | while read TUT; do \
+	    $(VENV_PATH)/bin/python3.10 $$TUT >/dev/null ; \
+	    echo Succesfully ran $$TUT ; \
```

Comment:
Where is TUT defined?

Reply:
just before line end near "... `while read TUT` ...". The previous XARGS variable evaluation was wrong and crashed when I ran it (each command runs in a separate shell).

Reply:
Another option would be to use a `find` command or a `<<$(..)` loop here-file construct instead, but it might be a bit too convoluted in a Makefile.